### DISCUSSION TITLE
Cannot resolve symbol @QueryMapping, @SchemaMapping, @Annotation

### DISCUSSION THREAD

**Discussion by CryptoGamer8** [graphql-java/graphql-java#3274] [Q&A]
Hi,

I'm just a beginner, following the[ tutorial](https://www.graphql-java.com/tutorials/getting-started-with-spring-boot/) to build up my first GraphQL application. I'm using Gradle-Groovy, Java 11, Spring Boot 2.7.14 as my tool set. The problem is,  @QueryMapping, @SchemaMapping, @Annotation doesn't come within the package. And I don't know where to add them.

This is my plugins in the build.gradle. 
```
plugins {
	java
	id("org.springframework.boot") version "2.7.14"
	id("io.spring.dependency-management") version "1.0.15.RELEASE"
}
```

Thanks for any help!

**Comment by dondonz** [comment]
Hello @CryptoGamer8 , the annotations come from the Spring for GraphQL project which you will need to add separately as a dependency.

Please see our tutorial which will show you how to add the right dependencies and show you how to use the annotations. https://www.graphql-java.com/tutorials/getting-started-with-spring-boot

The tutorial's code is in this repo: https://github.com/graphql-java/tutorials
